operators/grpc_operators/python/client/entity_client_service.py

```diff
@@ -0,0 +1,160 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import logging
+
+import grpc
+from holoscan.conditions import BooleanCondition
+
+from holohub.grpc_operators import holoscan_pb2, holoscan_pb2_grpc
+
+
+class EntityClientService:
+    def __init__(
+        self,
+        server_address,
+        interrupt,
+        request_queue,
+        response_queue,
+        source_operator,
```

Comment:
type hints on these arguments would make this easier to understand (I had to check in `app_edge_single_fragment.py` to be sure what types these queues were)